src/test/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDaoITest.java

```diff
@@ -169,4 +169,37 @@ void addExistingWithNewPasswordTest() {
         assertEquals(2, jdbcUserDao.countAll());
     }
 
+    @Test
+    @DisplayName("Edit non-existent user in db")
+    void editNonExistentUserInDbTest(){
+        logger.info("editNonExistentUserInDbTest()");
+        String oldEmail = "non-existent-user@vinyl.com";
+        User user = new User();
+        user.setEmail("someNewEmail@vinyl.com");
+        user.setPassword("HASH4");
+        user.setSalt("");
+        user.setIterations(0);
+        user.setRole(Role.USER);
+        user.setStatus(true);
+
+        assertFalse(jdbcUserDao.edit(oldEmail, user));
+    }
+
+    @Test
+    @DisplayName("Edit non-existent user in db")
```

Comment:
Maybe existing?

Reply:
I think that 'non-existent' is also correct. Please, check it

Reply:
It is!